README.md

```diff
@@ -1 +1,68 @@
-# nodejs-boilerplate
\ No newline at end of file
+![commercetools-icon](https://github.com/commercetools/press-kit/blob/master/PNG/72DPI/CT%20logo%20horizontal%20RGB%2072dpi.png?raw=true)
+
+# Node.js boilerplate
+[![Travis Build Status][travis-icon]][travis]
+[![Codecov Coverage Status][codecov-icon]][codecov]
+[![David Dependencies Status][david-icon]][david]
+[![David devDependencies Status][david-dev-icon]][david-dev]
+
+Start your next [Node.js commercetools](http://commercetools.github.io/nodejs-tools/) project in seconds.
+
+Table of Contents
+=================
+
+  * [Node.js boilerplate](#nodejs-boilerplate)
+    * [Features](#features)
+    * [Quick Start](#quick-start)
+    * [Requirements](#requirements)
+    * [Usage](#usage)
+    * [Development](#development)
+    * [Guidelines](#guidelines)
+
+## Features
+- Babel: use new JavaScript features while keeping compatibility.
+- TravisCI: config and setup included.
+- GreenKeeper: keep your dependencies up-to-date.
+- Guidelines: includes a contributing document
+- Readme: includes a readme template
+
+## Quick Start
+1. Clone this repo using `git clone --depth=1 https://github.com/commercetools/nodejs-boilerplate.git`
+2. Run `rm -r .git && npm run install` to clean the git repo and install dependencies.
```

Comment:
I think we can provide a bash script to prepare a "clean" installation of the boilerplate (e.g. that removes files specific to the boilerplate, install deps, etc).